GunShipAlpha/src/Game.java

```diff
@@ -122,12 +144,24 @@ private record spawnEnemiesRandomLocation(int x, int y) {
 
     @Override
     public void paint(Graphics g) {
-        int canvasWidth = getWidth();
-        int canvasHeight = getHeight();
-        int spriteX = (canvasWidth - sprite.getWidth()) / 2;
-        int spriteY = (canvasHeight - sprite.getHeight()) / 2;
-        g.drawImage(sprite, spriteX, spriteY, null);
-        render(g);
+        if (!gameOver) {
+            int canvasWidth = getWidth();
+            int canvasHeight = getHeight();
+            int spriteX = (canvasWidth - sprite.getWidth()) / 2;
+            int spriteY = (canvasHeight - sprite.getHeight()) / 2;
+            g.drawImage(sprite, spriteX, spriteY, null);
```

Comment:
i think this is painting the player, right? that should move inside the player, and the player should own the sprite.

Reply:
fixed